addons/account_invoice_import/wizard/account_invoice_send_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="account_invoice_send_inherit_account_wizard_form">
+            <field name="name">account.invoice.send.form.inherited.xml</field>
+            <field name="model">account.invoice.send</field>
+            <field name="inherit_id" ref="account.account_invoice_send_wizard_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//div[@name='option_email']" position='after'>
+                    <div name="is_xml">
+                        <field name="is_xml" />
+                        <b><label for="is_xml"/></b>
+                    </div>
```

Comment:
Is `<field name="is_xml"/>` not enough?

Reply:
No because the label is on the left of the field. The same is done for the other fields in this view